src/index.scss

```diff
@@ -1,6 +1,9 @@
 $spacer: 1rem;
 $blue: #007db8;
 $white: #fff;
+$component-active-bg: #0A3055FF !default;
+$component-active-color: $white !default;
+$rounded-pill: 50rem  !default;
```

Comment:
This one is easy too!

https://github.com/openedx/paragon/blob/0077397963cdd38462df72071a658317981c926b/scss/core/_variables.scss#L476
```scss
$rounded-pill:                50rem !default;
```